docs/docs/tutorial/debugger.md

```diff
@@ -0,0 +1,15 @@
+---
+sidebar_position: 11
+---
+
+# Debugger    
+
+Debugger is on the left-sidebar of the app builder. In case of a query fails, the error logs can be viewed in a popover when
+the debugger icon is clicked.
+<img class="screenshot-full" src="/img/tutorial/debugger/debugger.gif" alt="ToolJet - Debugger" height="420"/>
+
+
+## Pin Debugger
+Pinning the popover can be helpful while debugging the queries and will remain open till it is unpinned.
```

Comment:
`queries and will remain -> queries. The debugger will remain open until it is unpinned.